src/services/base/schema.graphql

```diff
@@ -0,0 +1,25 @@
+interface awsBaseService
+  @generate(
+    query: { get: false, query: false, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+  id: String! @id @search(by: [hash])
+  arn: String! @id @search(by: [hash])
+  accountId: String! @search(by: [hash, regexp])
+  region: String @search(by: [hash, regexp])
+  cgId: String @search(by: [hash])
```

Comment:
[This ](https://github.com/cloudgraphdev/cloudgraph-provider-aws/blob/main/src/services/cloudtrail/schema.graphql#L3) is why. As I told you, it is an uncommon case, but it exists. It appears so infrequently that it is better to leave it that way instead of creating a new base interface that only we will use once or twice.

Reply:
oh, I see! thanks!

Reply:
this cg id for cloudtrail causes us a lot of issues. I wonder if it makes sense to reconsider our solution here. 🤔 Idk what we could do outside of limiting cloudtrails to only show up in the account they belong to. (I believe this is only an issue for cloudtrails in multi accounts)